src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -514,6 +515,7 @@ class DialogmoteService(
                     newReferat = newReferat,
                     pdfId = null,
                     digitalt = true,
+                    sendAltinn = null,
```

Comment:
Synes det ser penere it med sendAltinn = false

Reply:
Enig, endret til false.